app/models/role.rb

```diff
@@ -30,13 +30,25 @@ def organisations
     links["ordered_parent_organisations"]
   end
 
+  def current_holder
+    links["role_appointments"]
+      .select { |ra| ra.dig("details", "current") == true }
```

Comment:
```suggestion
      .select { |ra| ra.dig("details", "current") }
```

Since it's already a boolean, you won't need this!

Reply:
Of course! 😆